packages/gamut-templates/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export { default as Interstitial } from './Interstitial';
 export { default as GridForm } from './GridForm';
 export { default as SplitInterstitial } from './SplitInterstitial';
+export { default as Modal } from './Modal';
```

Comment:
```suggestion
export { default as Modal } from './Modal';
export { default as SplitInterstitial } from './SplitInterstitial';
```

Nit: alphabetical order? 😄 

_(this is so incredibly far away from being blocking...)_

Reply:
Also whoa I just realized these should probably be `export * as`, as with Gamut... maybe that's why our `yarn start`s are being annoying? I'll try it out.

Reply:
https://github.com/Codecademy/client-modules/pull/689

Reply:
I used to like alphabetizing existing imports and props where order didn't matter (sometimes it matters in CSS), but I stopped because I think it makes the diffs a little less readable.